lib/services.rb

```diff
@@ -7,4 +8,8 @@ def self.publishing_api
       bearer_token: ENV['PUBLISHING_API_BEARER_TOKEN'] || 'example'
     )
   end
+
+  def self.rummager_api
```

Comment:
I think `Services.rummager` instead of `Services.rummager_api` would be more consistent with other applications.